### PR TITLE
add postgresql

### PR DESCRIPTION
**Pull request by TomasTomecek**
tested locally and it werks

### CODE REVIEW

File: openshift/pg.yml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        - name: POSTGRESQL_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              key: database-password
+              name: postgresql
+        - name: POSTGRESQL_DATABASE
+          valueFrom:
+            secretKeyRef:
+              key: database-name
+              name: postgresql
```
Comment (jpopelka) [inline openshift/pg.yml]:
```yaml
key: postgres-secret
name: database-name
```

File: playbooks/deploy.yml [hunk 2]
```diff
@@ -93,6 +96,7 @@
       loop:
         - ../openshift/redis.yml
         - ../openshift/service.yml
+        - ../openshift/pg.yml
```
Comment (jpopelka) [inline playbooks/deploy.yml]: aha, you've saved some letters, nice work  ;-)

File: openshift/pg.yml (hunk 1 above)
Comment (TomasTomecek) [reply openshift/pg.yml]: yup, just ran into this while playing with it locally

File: playbooks/deploy.yml (hunk 2 above)
Comment (TomasTomecek) [reply playbooks/deploy.yml]: yah, I'm lazy :D

File: openshift/pg.yml (hunk 1 above)
Comment (jpopelka) [reply openshift/pg.yml]: yeah, and I switched the keys  :man_facepalming:

**Pull request merged**